src/app/home/modules/about/containers/about/about.component.html

```diff
@@ -24,8 +24,10 @@
 
 			<ion-item>
 				<ion-icon name="pin" item-left></ion-icon>
-				Holiday Inn - Ghent Expo<br />
-				Maaltekouter 3, 9051 Ghent, Belgium
+				<a href="https://ng-be.org/location" target="_blank">
```

Comment:
Maybe we should make [`<ion-item>` clickable instead](https://ionicframework.com/docs/api/item#clickable-items)?

```html
<ion-item href="https://ng-be.org/location" target="_blank">
	<ion-icon name="pin" item-left></ion-icon>
	Holiday Inn - Ghent Expo<br />
	Maaltekouter 3, 9051 Ghent, Belgium
</ion-item>
```

Reply:
Adding an `href` on anything else then `<a>` is a no-go

Reply:
Not sure why they recommend such a bad thing

Reply:
The `href` attribute is just [an input to the component](https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/item/item.tsx#L100). This makes the component clickable and the [click is handled internally](https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/item/item.tsx#L390). So not sure if this is a bad thing? Unless `href` keyword has a special meaning in terms of a11y?